apps/ima/views/save-and-exit.html

```diff
@@ -1,5 +1,9 @@
 {{<partials-page}}
   {{$page-content}}
     <p>{{#t}}pages.save-and-exit.paragraph1{{/t}}</p>
+
+    <div class="other-details">
```

Comment:
other-details class has no styling in css so this div and class is not needed and can be removed